CMakeLists.txt

```diff
@@ -44,6 +44,12 @@ set(ENABLE_ASTYLE      OFF CACHE BOOL "")
 set(ENABLE_CLANGQUERY  OFF CACHE BOOL "")
 set(ENABLE_UNCRUSTIFY  OFF CACHE BOOL "")
 
+if (ENABLE_CUDA AND ON_BLUEOS)
+    # Very specific fix for working around CMake adding implicit link directories returned by the BlueOS
+    # compilers to link CUDA executables 
+    set(BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE "/usr/tce/packages/gcc/gcc-4.9.3/lib64/gcc/powerpc64le-unknown-linux-gnu/4.9.3;/usr/tce/packages/gcc/gcc-4.9.3/lib64" CACHE STRING "")
```

Comment:
I'd like to keep all this machine specific stuff in one place.  Can you move this into the spack package then have it generated into host-config?  I'd also be concerned this wouldn't get loaded early enough for when the checks are called (though i assume you tested this).

Reply:
That variable just needs to be set before BLT is included (as that's when all the setup stuff happens).

I had initially had this set in the serac `package.py` but changed it to avoid (sort of) introducing an explicit BLT dependency in the Spack config stuff.  If that's not a concern I'll move it back.

Reply:
definitely not a concern.